configure.ac

```diff
@@ -154,7 +154,7 @@ dnl Checks for library functions.
 dnl the *_r functions are in posix so we can use them unconditionally, but the ext/yahttp code is
 dnl using the defines.
 AC_CHECK_FUNCS_ONCE([strcasestr localtime_r gmtime_r recvmmsg sched_setscheduler])
-AC_CHECK_FUNCS_ONCE([getrandom getentropy arc4random arc4random_uniform arc4random_buf])
```

Comment:
May I suggest reusing `PDNS_CHECK_SECURE_MEMSET`, which is defined in `m4/pdns_check_secure_memset.m4` and used by at least dnsdist? It already checks `explicit_bzero` and `explicit_memset` (which seems forgotten here?) but not yet `memset_s`